### PR TITLE
Quickstart & Guide updates

### PR DESCRIPTION
**Pull request by Sanne**
- https://hibernate.atlassian.net/browse/OGM-921
 - https://hibernate.atlassian.net/browse/OGM-990
 - https://hibernate.atlassian.net/browse/OGM-991

### CODE REVIEW

File: documentation/manual/src/main/asciidoc/en-US/modules/getting-started.asciidoc [hunk 1]
```diff
@@ -2,20 +2,19 @@
...
 
-If you are familiar with JPA, you are almost good to go :-)
+If you are familiar with JPA, you are almost good to go.
 We will nevertheless walk you through the first few steps of persisting
 and retrieving an entity using Hibernate OGM.
 
 Before we can start, make sure you have the following tools configured:
 
-* Java JDK 7 or above
+* Java JDK 8
```
Comment (gunnarmorling) [inline documentation/manual/src/main/asciidoc/en-US/modules/getting-started.asciidoc]: Can we say in some way that JDK 7 is enough for other backends?

File: documentation/manual/src/main/asciidoc/en-US/modules/getting-started.asciidoc [hunk 2]
```diff
@@ -143,11 +143,16 @@ The final list of dependencies should look like this:
         <artifactId>jboss-transaction-api_1.2_spec</artifactId>
     </dependency>
 
-    <!-- JBoss Transactions dependency - this (or another implementation) is
-         provided in a Java EE container -->
+    <!-- Add the Narayana Transactions Manager
+         an implementation would be provided in a Java EE container,
+         but this works nicely embedded in JavaSE as well -->
```
Comment (gunnarmorling) [inline documentation/manual/src/main/asciidoc/en-US/modules/getting-started.asciidoc]: s/embedded//
s/JavaSE/Java SE/

File: documentation/manual/src/main/asciidoc/en-US/modules/getting-started.asciidoc (hunk 1 above)
Comment (DavideD) [inline documentation/manual/src/main/asciidoc/en-US/modules/getting-started.asciidoc]: It's written below

File: documentation/examples/gettingstarted/src/main/resources/infinispan-local.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  -->
+
+<!--
+    This is the testing configuration, running in LOCAL clustering mode to speedup tests.
+    Not all tests use this, so that at least one still validates the configuration
+    validity which would be shipped with a release (like JPAStandaloneTest).
+-->
+<infinispan
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="urn:infinispan:config:7.0 http://www.infinispan.org/schemas/infinispan-config-7.0.xsd"
```
Comment (gunnarmorling) [inline documentation/examples/gettingstarted/src/main/resources/infinispan-local.xml]: They didn't change to a new version 8.0 in ISPN 8?

File: documentation/examples/gettingstarted/src/main/java/org/hibernate/ogm/examples/gettingstarted/DogBreedRunner.java [hunk 4]
```diff
@@ -64,23 +62,4 @@ public static void main(String[] args) {
 
 	}
 
-	public static TransactionManager getTransactionManager() {
```
Comment (gunnarmorling) [inline documentation/examples/gettingstarted/src/main/java/org/hibernate/ogm/examples/gettingstarted/DogBreedRunner.java]: +100 :)

**Comment by gunnarmorling** [comment]
Added some comments. Feel free to merge yourself then. Thanks!

File: documentation/examples/gettingstarted/src/main/resources/infinispan-local.xml (hunk 3 above)
Comment (Sanne) [inline documentation/examples/gettingstarted/src/main/resources/infinispan-local.xml]: yes, but it's the same format.

**Comment by Sanne** [comment]
thanks. Applied the suggestions and merged.

**Pull request closed without merging**